src/settings/CheckoutSettings/CheckoutSettingsForm.test.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import CheckoutSettingsForm, {
+  getCustomFieldPatronIdentifiers,
+} from './CheckoutSettingsForm';
+
+jest.mock('@folio/stripes/core', () => ({
+  useCustomFields: jest.fn(() => []),
+}));
```

Comment:
Looks like it is not necessary here. You already added this mock to `test/jest/__mock__/stripesCore.mock.js` file.

Reply:
removed